adam/ontology/phase1_ontology.py

```diff
@@ -66,6 +92,120 @@ def subtype(sub: OntologyNode, _super: OntologyNode) -> None:
 # learner.
 ON = OntologyNode("on")
 subtype(ON, SPATIAL_RELATION)
+PART_OF = OntologyNode("partOf")
+"""
+A relation indicating that one object is part of another object.
+"""
+subtype(PART_OF, RELATION)
+
+SIZE_RELATION = OntologyNode("size-relation")
+subtype(SIZE_RELATION, RELATION)
+
+BIGGER_THAN = OntologyNode("biggerThan")
+"""
+A relation indicating that one object is bigger than another object.
+
+This is a placeholder for a more sophisticated representation of size:
+https://github.com/isi-vista/adam/issues/70
+"""
+subtype(BIGGER_THAN, SIZE_RELATION)
+
+SMALLER_THAN = OntologyNode("smallerThan")
+"""
+A relation indicating that one object is smaller than another object.
+
+This is a placeholder for a more sophisticated representation of size:
+https://github.com/isi-vista/adam/issues/70
+"""
+subtype(SMALLER_THAN, SIZE_RELATION)
+
+
+def bigger_than(obj1: SubObject, obj2: SubObject) -> Tuple[SubObjectRelation, ...]:
+    return (
+        SubObjectRelation(BIGGER_THAN, obj1, obj2),
+        SubObjectRelation(SMALLER_THAN, obj2, obj1),
+    )
+
+
+SUPPORTS = OntologyNode("supports")
+"""
+A relation indicating that  one object provides the force to counteract gravity and prevent another 
+object from falling.
+"""
+subtype(SUPPORTS, SPATIAL_RELATION)
+
+
+def supports(obj1: SubObject, obj2: SubObject) -> Tuple[SubObjectRelation, ...]:
+    """
+    Define a support `SubObjectRelation` for `HierarchicalObjectSchema`
```

Comment:
```
Convenience method for indicating that one `SubObject` in a `HierarchicalObjectSchema` has a `SUPPORTS` relation with another.

For us with `sub_object_relations`.
```

Note the return is not accurate - it does not return a `SubObjectRelation` but a *tuple* wrapping a single `SubObjectRelation`. But I think if we point the user to `sub_object_relations` we can omit the documentation of the returns.

Reply:
(please alter the others in a similar fashion)